packages/theme-light-extension/style/variables.css

```
@@ -412,4 +412,9 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-jupyter-icon-color: var(--md-orange-900);
   --jp-notebook-icon-color: var(--md-orange-700);
   --jp-svg-icon-color: var(--md-orange-700);
+  --jp-console-icon-color: var(--md-blue-700);
+  --jp-terminal-icon-color: var(--md-grey-800);
+  --jp-terminal-icon-background_color: var(--md-grey-200);
```

The convention is to use dash as separator:

```suggestion
  --jp-terminal-icon-background-color: var(--md-grey-200);
```

---

This has been corrected in https://github.com/jupyterlab/jupyterlab/pull/13279/commits/87d925f82e1a371d202d8832855287c8b63dd634.